src/GafferScene/Render.cpp

```diff
@@ -255,6 +255,11 @@ void Render::execute() const
 	RendererAlgo::outputLights( adaptedInPlug(), globals.get(), renderSets, renderer.get() );
 	RendererAlgo::outputObjects( adaptedInPlug(), globals.get(), renderSets, renderer.get() );
 
+	if( renderScope.sceneTranslationOnly() )
```

Comment:
This is causing the build to fail.

```
src/GafferScene/Render.cpp: In member function 'virtual void GafferScene::Render::execute() const':
src/GafferScene/Render.cpp:258:6: error: 'renderScope' was not declared in this scope
  if( renderScope.sceneTranslationOnly() )
      ^
```

This commit is building on top of previous changes that weren't backported, because they're part of the 0.54 performance effort. I think we can just drop it from the backport.

Reply:
Dropped